neMag/Models/Category.cs

```diff
@@ -11,13 +11,18 @@ public class Category
     {
         [Key]
         public int CategoryId { get; set; }
+        
+        public int ParentId { get; set; }
 
         [Required(ErrorMessage = "Title is mandatory!")]
         public string Title { get; set; }
 
         [Required(ErrorMessage = "Description is mandatory!")]
         public string Description { get; set; }
 
+        // What's this? I think we can safely remove it
```

Comment:
I agree. I think it's one of Florin's early tests to check that the database migration was working correctly.